include/species.hpp

```diff
@@ -14,6 +14,8 @@ class Species;
 
 class Species {
 public:
+  Species(); // unused but required by cppyy python bindings
```

Comment:
This seems to be causing the integration test `Electrostatic2D3V.TwoStream` to fail. Not entirely sure why that would happen, but when I comment it out the test passes.